include/tvm/ir/expr.h

```diff
@@ -165,6 +166,20 @@ class RelayExprNode : public BaseExprNode {
   template <typename TTypeNode>
   inline const TTypeNode* type_as() const;
 
+  /*!
+   * \brief The storage execution scope (SEScope) for this node (the result of device planning).
```

Comment:
Extra color for the comment, feel free to reword:

For first-order (ie non function) expressions this describes where the result of evaluating the expression should be stored. Note that currently all composite first-order values (tuples, references, ADTs) must be stored on the same SEScope. Eg it is not currently possible to store two tuple fields on different devices. Thus a single SEScope sufficies for those types. 

For expressions of function type this describes where the result of calling the function should be stored (as opposed to, say, where the function or closure itself is stored). The SEScope's Target also describes how the body of the function should be compiled.